src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Authoring.Tests/Microsoft.Azure.CognitiveServices.LUIS.Authoring.Tests.csproj

```diff
@@ -9,6 +9,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
Do you need to add the `PrivateAssets` and `IncludeAssets` tags?

Reply:
@dsgouda done.